src/internal/modules/charge-information/forms/use-abstraction-data.js

```diff
@@ -35,10 +53,12 @@ const useAbstractionDataForm = request => {
   return f;
 };
 
-const useAbstractionDataSchema = () => {
+const useAbstractionDataSchema = (request) => {
+  const { chargeVersions } = request.pre;
+  const validIds = chargeVersions.map(cv => (cv.status === 'current' || cv.status === 'superseded') ? cv.id : null);
```

Comment:
I guess this might need tweaking if we tweak the above

Reply:
I think the filtering of the charge versions has gone from above so maybe not needed here either?

Reply:
of course thanks